shopping-page.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GitShop</title>
+    <link rel="stylesheet" href="./stylecss/shopping-page.css">
+</head>
+<body>
+   
+    <section class="page-container">
+    <img class="shop-img" src="./image/shopping-bags.jpg" alt="shopping |E-Commerce">
+    <div class="content">
+        
+        <span id = "titles">Our Vision <br> </span> <br />
+        <span  id ="term">" To build a place where people can come to find and discover anything <br>they  might want to buy online." <br></span> <br />
+   
+        <span id = "titles" >Our Mission <br> </span> <br />
+        <span  id ="term" > "Our mission is to continually raise the bar of the customer<br> experience by using the internet and technology to help <br>consumers find, discover and buy anything, and empower<br> businesses and content creators to maximise their success."</span
+        >
+       <section class="container">
+      <div id="customer">
+          
+            <h1> About Seller  </h1>
+            <p> GitShop believes in the importance of the customer, so we are keen to employ the best <br> provide quality with the highest standards, and we always think deeper to serve our customers</p>
```

Comment:
If you want, you could try a preformatted `<pre>` element instead of the `<p>` element and then you wouldn't have to insert the `<br>` tags yourself, because the browser will use the line breaks that you have in the text.

Here is the MDN page on `<pre>` elements:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre